client/packages/system/src/Stock/Components/InventoryAdjustment/InventoryAdjustmentDirectionSearchInput.tsx

```diff
@@ -1,38 +1,43 @@
-import React, { FC } from 'react';
+import React from 'react';
 import {
   AdjustmentTypeInput,
-  Autocomplete,
+  MinusIcon,
+  PlusIcon,
+  ToggleButtonGroup,
   useTranslation,
 } from '@openmsupply-client/common';
 
 interface InventoryAdjustmentDirectionInputProps {
   value: AdjustmentTypeInput;
-  onChange: (type?: AdjustmentTypeInput) => void;
+  onChange: (type: AdjustmentTypeInput) => void;
 }
 
-export const InventoryAdjustmentDirectionInput: FC<
-  InventoryAdjustmentDirectionInputProps
-> = ({ value, onChange }) => {
+export const InventoryAdjustmentDirectionInput = ({
+  value,
+  onChange,
+}: InventoryAdjustmentDirectionInputProps) => {
   const t = useTranslation();
 
   const options = [
-    { label: t('label.increase-qty'), value: AdjustmentTypeInput.Addition },
-    { label: t('label.decrease-qty'), value: AdjustmentTypeInput.Reduction },
+    {
+      id: 'decrease',
+      label: t('label.decrease-qty'),
+      value: AdjustmentTypeInput.Reduction,
+      icon: <MinusIcon />,
+    },
+    {
+      id: 'increase',
+      label: t('label.increase-qty'),
+      value: AdjustmentTypeInput.Addition,
+      icon: <PlusIcon />,
+    },
```

Comment:
Bit of a usability thing -- at first I thought it was broken cos it looked like I should be able to click the "+" and "-" buttons to increase or decrease the number. I didn't realise it was just to set the number as positive or negative. Is it clear enough, or is it just me?

Maybe the missing gap I mentioned earlier might help?